README.md

```diff
@@ -1,3 +1,6 @@
+> [!WARNING] 
+> SECURITY FIXES ONLY: This project is not actively maintained. All other contributions will not be accepted.
```

Comment:
```suggestion
> [!Attention] 
> Mozilla has archived this repository because both the protocol and documentation have reached completion, with no further updates or changes necessary.
> 
> While the existing code will remain available for reference or fork, no further updates or support should be expected.
```
Changed to match the wording of the reasons given in [the issue](https://github.com/mozilla/hawk/issues/293#issuecomment-2032185977)